docs/design/2020-06-24-placement-rules-in-sql.md

```diff
@@ -283,41 +314,71 @@ When the `CONSTRAINTS` option doesn't contain `count`, `REPLICAS` must be specif
 
 `REPLICAS` in the statement is equivalent to field `count` in PD placement rule configuration.
 
-### Key range configuration
+#### Key range configuration
 
 In PD placement rule configuration, the key range must be specified. Now that `table_name` is specified in the `ALTER TABLE` statement, key range can be inferred.
 
 Typically, key format is in such a format: `t_{table_id}_r_{pk_value}`, where `pk_value` may be `_tidb_rowid` in some cases. `table_id` can be inferred from `table_name`, thus key range is `t_{table_id}_` to `t_{table_id+1}_`.
 
-Similarly, key range of partitions and indices can also be inferred.
+Similarly, key range of partitions can also be inferred.
```

Comment:
Since `indices` is removed here, maybe all mentioning of indices/index/indexes should be removed from the whole document or clarified that indexes are not included in this document for placement policies, (other than they will end up where the database/table/partition is placed)

Reply:
I believe I've removed it from everywhere now, thanks!